app/Http/Controllers/TransactionController.php

```diff
@@ -21,8 +23,13 @@ public function export(Transaction $transaction)
         return view('transactions.export', compact('transaction'));
     }
 
-    public function duplicate(Transaction $transaction)
+    public function duplicate($uuid)
     {
+        $transaction = Transaction::where('uuid', $uuid)->first();
```

Comment:
@ZeemZach this would work, but there's a better approach: see #1 Pull Request

Reply:
Got It  :)